modules/utxo-staking/src/babylon/delegationMessage.ts

```diff
@@ -72,16 +73,44 @@ export function getBtcProviderForECKey(
     return psbt;
   }
 
+  function signBip322Simple(message: string): string {
+    // Get the script public key from the staking descriptor
+    const scriptPubKey = Buffer.from(descriptorBuilder.getStakingDescriptor().scriptPubkey());
+
+    // Build to_spend transaction for BIP-322
+    const toSpendTx = BIP322.buildToSpendTx(message, scriptPubKey);
+
+    // Get the to_spend txid
+    const toSpendTxId = toSpendTx.getId();
+
+    // Build to_sign transaction
+    const toSignPsbt = BIP322.buildToSignTx(toSpendTxId, scriptPubKey);
```

Comment:
buildToSignTx takes in a tapInternalKey as the last parameter, should we pass that in (basically the no-op pub key used for babylon staking)? Can you please double check?

Reply:
yeah this was all wrong